en/prepare/attributes-from.rst

```diff
@@ -33,7 +33,7 @@ The field widgets from QGIS are supported on a best effort basis.
 +-------------------+-----------------+-------------------------------------------------+
 | Hidden            | :yay:`✔`        |                                                 |
 +-------------------+-----------------+-------------------------------------------------+
-| External Resource | :moreorless:`~` | This field is combined with camera integration. |
+| Attachment | :moreorless:`~` | This field is combined with camera integration. |
```

Comment:
Can you fix the indentation here? There are spaces missing to make the vertical lines look like a table.